druid/examples/styled_text.rs

```diff
@@ -22,7 +22,8 @@ use druid::{
 use std::fmt::Display;
 
 // This is a custom key we'll use with Env to set and get our font.
-const MY_CUSTOM_FONT: Key<FontDescriptor> = Key::new("styled_text.custom_font");
+const MY_CUSTOM_FONT: Key<FontDescriptor> =
+    Key::new("org.linebender.druid.theme.styled_text.custom_font");
```

Comment:
This isn't part of the theme, so I'd probably just do something like:

```suggestion
    Key::new("org.linebender.example.my-custom-font");
```